src/storage/src/hummock/file_cache/cache.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022 Singularity Data
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::Arc;
+
+use risingwave_common::cache::LruCache;
+
+use super::coding::CacheKey;
+use super::error::Result;
+use super::filter::Filter;
+use super::meta::SlotId;
+use super::store::{Store, StoreOptions, StoreRef};
+
+pub struct FileCacheOptions {
+    pub dir: String,
+    pub capacity: usize,
+    pub cache_file_fallocate_unit: usize,
+    pub filters: Vec<Arc<dyn Filter>>,
```

Comment:
What is the `filters` used for?

Reply:
It's for user to customize some rule to judge if the key can be inserted.